Apps/Src/Voltage.c

```diff
@@ -99,7 +99,14 @@ SafetyStatus Voltage_CheckStatus(void){
  */
 SafetyStatus *Voltage_GetModulesInDanger(void){
 	static SafetyStatus checks[NUM_BATTERY_MODULES];
-	uint32_t open_wires = Voltage_GetOpenWire();
+	Voltage_GetOpenWire();
+	uint32_t open_wires = 0;
+
+	//put all the bits from each minion's system_open_wire variable into one variable
+	for(int k = 0; k < NUM_MINIONS; k++){
+		open_wires += Minions[k].system_open_wire & 0x0FF;	//there are at most 8 modules per IC 
```

Comment:
Bit 0 is pin C0 which is GND. Bit 1 would be the wire between modules 1 and 2, bit 2 would be the wire between modules 2 and 3, and so on (battery modules 1-indexed).
```suggestion
		open_wires += Minions[k].system_open_wire & 0x01FF;	//there are at most 8 modules per IC 
```